captum/attr/_core/layer/grad_cam.py

```diff
@@ -82,6 +82,7 @@ def attribute(
         additional_forward_args: Any = None,
         attribute_to_layer_input: bool = False,
         relu_attributions: bool = False,
+        split_channels: bool = False,
```

Comment:
@dzenanz, `channel` suffix seems to be image specific. Our implementation is generic and we prefer having generic names. Do you mind giving a more generic name ? Something like: `attr_dim_summation=False` ? 
cc: @vivekmig

Reply:
Done.

Reply:
Now I realize this rename would need to invert the logic, and the default from False to True. I will force-push.

Reply:
Thank you very much for the quick fix, @dzenanz! I'll let @vivekmig to have a final look.